caput/tests/conftest.py

```diff
@@ -124,6 +124,25 @@ def write_output(self, filename, output, file_format=None, **kwargs):
   style: 'fried'
 """
 
+multi_eggs_pipeline_conf = """
+---
+pipeline:
+  tasks:
+    - type: caput.tests.conftest.GetEggs
+      params: eggs_params
+      out: [color, egg]
+    - type: caput.tests.conftest.CookEggs
+      params: cook_params
+      in: egg
```

Comment:
```suggestion
      in: color
```
Not 100% sure if I understood the intention of this test. Were the same eggs supposed to be fried twice? If the idea was to test a task with multiple outputs, where the outputs get used by subsequent tasks, then I think `color` should be an input somewhere.